boards/arm/mimxrt1050_evk/mimxrt1050_evk.dts

```diff
@@ -38,6 +40,22 @@
 		device_type = "memory";
 		reg = <0x80000000 0x2000000>;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led: led@0 {
+			gpios = <&gpio1 9 GPIO_INT_ACTIVE_LOW>;
```

Comment:
Why are we consistently using `GPIO_INT_ACTIVE_LOW ` when specifying LED GPIO polarity? Shouldn't we be using `GPIO_POL_NORMAL` and `GPIO_POL_INV` instead?